src/utils/hooks/api/use-get-collateral-currencies-data.tsx

```diff
@@ -0,0 +1,61 @@
+import { CollateralCurrencyExt, CurrencyExt } from '@interlay/interbtc-api';
+import { Currency, ExchangeRate } from '@interlay/monetary-js';
+import Big from 'big.js';
+import { useQuery, UseQueryResult } from 'react-query';
+
+import { useGetCollateralCurrencies } from './use-get-collateral-currencies';
+
+type CollateralCurrenciesData = { [currencyTicker: string]: CollateralCurrencyData };
+
+type CollateralCurrencyData = {
+  liquidationThreshold: Big;
+  secureThreshold: Big;
+  exchangeRate: ExchangeRate<Currency, CurrencyExt>;
+};
+
+const getCollateralCurrencyData = async (collateralToken: CollateralCurrencyExt): Promise<CollateralCurrencyData> => {
+  try {
```

Comment:
Is try/catch necessary here? `useQuery` should handle returning the errors for you.

Reply:
Good idea :+1: